tests/resources/test_issue.py

```diff
@@ -26,6 +26,20 @@ def test_issue(self):
         self.assertEqual(issue.key, self.issue_1)
         self.assertEqual(issue.fields.summary, "issue 1 from %s" % self.project_b)
 
+    def test_issue_get_field(self):
+        issue = self.jira.issue(self.issue_1)
+        self.assertEqual(
+            issue.fields.description, issue.get_field(fieldName="description")
+        )
+
+        self.assertRaisesRegex(
+            AttributeError, ": _something", issue.get_field, "_something"
+        )
+
+        self.assertRaisesRegex(
+            AttributeError, "customfield_1234", issue.get_field, "customfield_1234"
+        )
```

Comment:
(non-blocking) I have a preference for the [context manager style](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertRaisesRegex) as I find it easier to read!

```suggestion
        with self.assertRaisesRegex(AttributeError, ": _something"):
            issue.get_field("_something")

        with self.assertRaisesRegex(AttributeError, "customfield_1234"):
            issue.get_field("customfield_1234")
```